app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -111,6 +135,10 @@ export default {
     },
     submissionId: String,
     versionId: String,
+    isDuplicate: {
```

Comment:
isDuplicate flag checked if the submission is a submission created from an existing one or not? If this flag is **TRUE** then this submission will be created as a New submission else it will be considered an Update to the existing submission. The motive is to use this flag using the same View (code).
So by clicking on "**Copy from existing submission**" when a Submitter redirects to the page then this isDuplicate flag will be marked TRUE, which Means the submitter is creating a submission from an existing one. On the other hand, if a user wants to update an existing submission then this flag will remain FALSE, and functionality act as usual for updating a submission.

Reply:
Is there any way to disable this duplicate submissions feature, disabled by default? We have users that don't want submitters to duplicate data as they will just duplicate a submission without updating the values.